src/cookbooks/normandy_events.md

```diff
@@ -73,24 +50,24 @@ The `event_map_values` column of unenroll events includes a `reason` key.
 Reasons are described in the [Normandy docs][normandy-doc].
 Normal unenroll events at the termination of a study will occur for the reason `recipe-not-seen`.
 
-To fetch a count of events by reason and branch in PySpark:
-
-```python
-unenrollments_by_reason = (
-  events
-  .filter(events.event_category == "normandy")
-  .filter(events.event_method == "unenroll")
-  .filter(events.event_string_value == EXPERIMENT_SLUG)
-  .filter(events.submission_date_s3 >= EXPERIMENT_START)
-  .withColumn("branch", events.event_map_values.getItem("branch"))
-  .withColumn("reason", events.event_map_values.getItem("reason"))
-  .groupBy(events.submission_date_s3, "branch", "reason")
-  .agg(f.count("*").alias("n"))
-  .toPandas()
-)
+To fetch a count of events by reason and branch:
 
+```sql
+SELECT
+  submission_date,
+  udf.get_key(event_map_values, 'branch') AS branch,
+  udf.get_key(event_map_values, 'reason') AS reason,
+  COUNT(*) AS n
+FROM telemetry.events
+WHERE
+  event_category = 'normandy'
+  AND event_method = 'unenroll'
+  AND event_string_value = '{{experiment_slug}}'
+  AND submission_date >= '{{experiment_start}}'
+GROUP BY 1, 2, 3
+ORDER BY 1, 2, 3
 ```
 
 [normandy-doc]: https://firefox-source-docs.mozilla.org/toolkit/components/normandy/normandy/data-collection.html#enrollment
-[`telemetry_shield_study_parquet`]: https://docs.telemetry.mozilla.org/datasets/shield.html#telemetry_shield_study_parquet
+[`shield_study`]: https://docs.telemetry.mozilla.org/datasets/shield.html#telemetryshield_study
```

Comment:
This should probably not reference dtmo directly for local development. Haven't tested but I think this should work:
 
```suggestion
[`shield_study`]: ../datasets/shield.md#telemetryshield_study
```